ouroboros-consensus/src/Ouroboros/Consensus/Node.hs

```diff
@@ -718,6 +721,7 @@ initNetworkLayer :: forall m up hdr blk.
                     , HasHeader hdr
                     , hdr ~ blk     -- TODO: for now
                     , Eq blk        -- TODO: for now
+                    , Show blk
```

Comment:
Why do we need to add `Show blk`?

Reply:
If `sayTracer` (https://github.com/input-output-hk/ouroboros-network/pull/298/files#diff-a0e24f461e7fb4d85f24eba52bc9ae39R23) is used then we will have to set this constraint.

Reply:
Will remove this also.